src/domains/dashboard/components/chart-wrapper.tsx

```diff
@@ -43,7 +43,7 @@ export const ChartWrapper = ({
   //   ? { width: "100%", height: `${height}px` }
   //   : { width: "100%" }
 
-  const chartUuid = useMemo(() => uuid.v4(), [])
+  const chartUuid = useMemo(() => getUuid.v4(), [])
```

Comment:
IMHO changing name like this doesn't bring value, if the former way is already used in other places, and it's remembered